gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -40,8 +42,10 @@ from google.api_core import future
 from google.api_core import operation
```

Comment:
PR #1094 also added this line. If it’s only used for rest transports, we should also guard it with

```
 {% if "rest" in opts.transport %}
```